### DISCUSSION TITLE
Will there be an option to download argon (pro) skin so user can manually edit it?

### DISCUSSION THREAD

**Discussion by crumpledmint** [ppy/osu#21877] [Q&A] [answered]
Title

**Comment by mk56-spn** [comment, marked as answer]
That's not how it works argon isn't like stable skins. It's hard coded into the game using new techniques. Whilst it will probably be possible to export it at some point it is fundamentally different from all skins up till now ( except for triangles ) . As things currently stand argon can only be changed from the Dev side. You can of course compile your own lazer binary with changes to argon , but you won't be able to play on the official servers
